nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/work/StreamWorkManager.java

```diff
@@ -204,8 +269,9 @@ public void init() {
             index();
             initTopology();
             logManager = getLogManager();
-            streamProcessor = new LogStreamProcessor(logManager);
-            streamProcessor.init(topology, settings);
+            streamManager = getStreamManager();
+            streamManager.register("SWMDisable", topologyDisabled, settings);
+            streamProcessor = streamManager.registerAndCreateProcessor("SWM", topology, settings);
```

Comment:
Consider using full names `"SWM"` -> `"StreamWorkManager"` and  `"SWMDisable"` -> `"StreamWorkManagerDisable"`